src/huntsman/pocs/scheduler/observation.py

```diff
@@ -118,3 +118,22 @@ def __init__(self, position, pattern=dither.dice9, n_positions=9,
 
         # Specify directory root for file storage
         self._directory = os.path.join(self._image_dir, 'flats')
+
+    def get_exposure_filename(self, camera):
+        """ Get the exposure filename for a camera.
+        Args:
+            camera (Camera): A camera instance.
+        """
+        path = os.path.join(self.directory, camera.uid, self.seq_time)
+        filename = os.path.join(
+            path, f'flat_{self.current_exp_num:02d}.{camera.file_extension}')
```

Comment:
We could go above 99 in a sequence.

```suggestion
            path, f'flat_{self.current_exp_num:03d}.{camera.file_extension}')
```